crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -203,6 +156,9 @@ impl Node for BloomNode {
         self.view_query.update_archetypes(world);
     }
 
+    // Atypically for a post-processing effect, we do not need to
+    // use a secondary texture normally provided by view_target.post_process_write(),
+    // instead we write into our own bloom texture and then directly back onto main.
```

Comment:
Is this because we're reading from/writing to different mip levels?

Reply:
Sorta. You only need post_process_write() when you want to both read and write from the view target in the _same_ pass. In this render node, we split the work over multiple render passes, first reading from the view target, then writing to some intermediary mips of a different texture, and then writing the result to the view target. So there's no need for post_process_write().